scenarios/turbine-heat-processor/gradle.properties

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
I think this is the right formatting for this file, no?

```
#
# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
```

Reply:
Done.